src/components/button-link/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export interface Properties {
+  url: string;
+  className?: string;
+  openInNewTab?: boolean;
+}
+
+export class ButtonLink extends React.Component<Properties, undefined> {
+  get href() {
+    const { url } = this.props;
+    let protocol = '';
+
+    if (!/^http[s]?:\/\/(.*)$/.test(url)) {
+      protocol = 'https://';
+    }
+
+    return `${protocol}${url}`;
+  }
+
+  renderLink() {
+    const linkProps: any = {
+      className: classNames('button-link', this.props.className),
+      href: this.href,
+    };
+
+    if (this.props.openInNewTab) {
+      linkProps.target = '_blank';
+      linkProps.rel = 'noreferrer noopener';
+    }
+
+    return (
+      <a {...linkProps}>
+        <div className='button-link__label'>
```

Comment:
not sure why we need `div` inside `a`

Reply:
@mrval5, here's the original component. This PR is a verbatim copy: https://github.com/m3m3n70/zero-web/blob/development/src/components/button-link/index.tsx